powerline-shell.go

```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/otann/powerline-go/powerline"
+	"github.com/baol/powerline-go/powerline"
```

Comment:
Well, I understand the purpose behind, but it doesn't makes sense in this fork ;)